### PR TITLE
Infrastructure as a Code Startup Solution

### PR DESCRIPTION
**Pull request by fericode**
Add Infrastructure as a Code Startup Solution article

### CODE REVIEW

File: docs/blog/infrastructure-as-a-code-startup.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+﻿## **Infrastructure as a Code** Startup Solution
```
Comment (ysmoradi) [inline docs/blog/infrastructure-as-a-code-startup.md:1]: What does startup mean here?

File: docs/blog/infrastructure-as-a-code-startup.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+This method, due to its automation, reduces the possibility of errors and, depending on the implementation method, can be very similar to Migrations in EntityFramework, as migrations are created and applied to the Development database, and only if the tests are passed, the changes can be sent automatically to the QA, etc, and it is not possible to forget anything in between.
+
+One of the best tools for `Infra as a code` is a tool called **[Pulumi](https://www.pulumi.com/)**, which is compatible with Kubernetes, Azure, AWS, and Google Cloud.
+
+**Kubernetes**, for example, has its own methods for maintaining the Infra structure in its configuration files, but Pulumi offers both simplicity and ease in the Cloud, where you typically use your own Cloud Database Services and App Services and Logging Systems which are not Kubernetes subsets. You can control the whole Cloud and Kubernetes at the same time with one tool.
+
+For example, people in the Cloud, use `Database as a service` instead of creating a database in Kubernetes, which means achieving higher quality and reducing costs. Some other examples are requesting DDoS protection service and CDN and Media Services.
```
Comment (ysmoradi) [inline docs/blog/infrastructure-as-a-code-startup.md:13]: Not all people uses database as a service, but most of them will

**Review by ysmoradi — CHANGES_REQUESTED** [review]

File: docs/blog/infrastructure-as-a-code-startup.md (hunk 1 above)
Comment (fericode) [reply docs/blog/infrastructure-as-a-code-startup.md:1]: I think it is a little vague. Better words may be Setting up or initiation?

File: docs/blog/infrastructure-as-a-code-startup.md (hunk 2 above)
Comment (fericode) [reply docs/blog/infrastructure-as-a-code-startup.md:13]: OK. I will fix it in the next commit

File: docs/blog/infrastructure-as-a-code-startup.md (hunk 1 above)
Comment (ysmoradi) [reply docs/blog/infrastructure-as-a-code-startup.md:1]: Just remove startup (":

Comment (fericode) [reply docs/blog/infrastructure-as-a-code-startup.md:1]: OK

**Review by ysmoradi — APPROVED** [review]

**Pull request merged**